src/renderer/components/+namespaces/namespace-select-filter-model/namespace-select-filter-model.tsx

```diff
@@ -64,34 +67,37 @@ export class NamespaceSelectFilterModel {
         )}
       </div>
     );
-  };
+  }
 
   readonly menuIsOpen = observable.box(false);
 
-  closeMenu = action(() => {
+  @action
+  closeMenu() {
     this.menuIsOpen.set(false);
-  });
+  }
 
-  openMenu = action(() => {
+  @action
+  openMenu(){
     this.menuIsOpen.set(true);
-  });
+  }
 
   get selectedNames() {
     return untracked(() => this.dependencies.namespaceStore.selectedNames);
   }
 
-  isSelected = (namespace: string | string[]) =>
-    this.dependencies.namespaceStore.hasContext(namespace);
+  isSelected(namespace: string | string[]) {
+    return this.dependencies.namespaceStore.hasContext(namespace);
+  }
 
-  selectSingle = (namespace: string) => {
+  selectSingle(namespace: string) {
     this.dependencies.namespaceStore.selectSingle(namespace);
-  };
+  }
```

Comment:
Hmm. Is there a reason behind this consolidation to methods over lambdas?

Reply:
Only if you think that using it point free should be the same. But note that this is equivalent anyway, because I use `autoBind(this)`.

Reply:
> Only if you think that using it point free should be the same.

This is not the same, as one adds to quality of code and one does not because e.g.:

I can see `autoBind(this)`, but that's another complication and reduces portability of code. Also the function syntax adds some noise, and communicates false significance for `this`.

Tooling should not hinder quality of code. I feel like there must be a better solution in **tweaking the tooling** itself, instead of code. I urge you to revert, but do resolve if you feel strongly about these methods for now. Let's just not make a policy about this, as it's just not good.

Reply:
How does it reduce portability? It actually increases portability because it allows you to use proper class hierarchy for methods, which field functions do not. I am fine with field functions in plain objects, but I don't think that they should be used in classes.

Reply:
So it's the resolve then :)